tests/unit/adapters/admin-statistics-event-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleFor, test } from 'ember-qunit';
```

Comment:
The tests are failing if we use unit-helper, I'm checking why they fail. Thanks for pointing it :)